src/plugins/visualizations/public/services.ts

```diff
@@ -109,10 +108,6 @@ export const [getSavedSearchLoader, setSavedSearchLoader] = createGetterSetter<S
   'savedSearchLoader'
 );
 
-export const [getSavedAugmentVisLoader, setSavedAugmentVisLoader] = createGetterSetter<
-  SavedAugmentVisLoader
->('SavedAugmentVisLoader');
-
```

Comment:
as mentioned in [this comment](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3961/files/d2e2a482ae6cd62f680bf8e067a83c8ed23b3d95..e179ef5ce07cb1331ee9b5164516d8f0370bd527#r1192801738), I think we should keep this service here

Reply:
I am curious why that is the best practice. `vis_augmenter` is the source of truth of creating this.

Reply:
Right - so vis augmenter already creates it and sets it as an available obj to use here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/feature/feature-anywhere/src/plugins/vis_augmenter/public/plugin.ts#L17

And then the idea is we expose the loader in `visualizations` by having the getters/setters for the loader set here. For example, the rest of the getters/setters within this file are all using exposed services from dependent plugins.

Reply:
Here is an example in the AD plugin:
public/plugin.ts: https://github.com/amitgalitz/anomaly-detection-dashboards-plugin/blob/associated-detectors-1/public/plugin.tsx#L68
public/services.ts: https://github.com/amitgalitz/anomaly-detection-dashboards-plugin/blob/associated-detectors-1/public/services.ts#L11